src/pages/ServiceDetails/ServiceDescription.tsx

```diff
@@ -0,0 +1,185 @@
+    const apps: string[] = [];
+    const workloads: AppWorkload[] = [];
+    if (this.props.serviceDetails) {
+      if (this.props.serviceDetails.workloads) {
+        this.props.serviceDetails.workloads.forEach(wk => {
+          if (wk.labels) {
+            const appName = wk.labels[serverConfig.istioLabels.appLabelName];
+            if (!apps.includes(appName)) {
+              apps.push(appName);
+            }
+          }
+          workloads.push({
+            workloadName: wk.name,
+            istioSidecar: wk.istioSidecar
+          });
+        });
+      }
+    }
+    // We will show service labels only when there is some label that is not present in the selector
+    let showServiceLabels = false;
+    if (
+      this.props.serviceDetails &&
+      this.props.serviceDetails.service.labels &&
+      this.props.serviceDetails.service.selectors
+    ) {
+      const keys = Object.keys(this.props.serviceDetails.service.labels);
+      for (let i = 0; i < keys.length; i++) {
+        const key = keys[i];
+        const value = this.props.serviceDetails.service.labels[key];
+        if (this.props.serviceDetails.service.selectors[key] !== value) {
+          showServiceLabels = true;
+          break;
+        }
+      }
+    }
+    const serviceProperties = (
+      <div key="properties-list" className={resourceListStyle}>
+        <ul style={{ listStyleType: 'none' }}>
+          {this.props.serviceDetails && (
+            <li>
+              <span>Created</span>
+              <div style={{ display: 'inline-block' }}>
+                <LocalTime time={this.props.serviceDetails.service.createdAt} />
+              </div>
+            </li>
+          )}
+          {this.props.serviceDetails && (
+            <li>
+              <span>Version</span>
+              {this.props.serviceDetails.service.resourceVersion}
+            </li>
+          )}
+          {this.props.serviceDetails &&
+            this.props.serviceDetails.additionalDetails &&
+            this.props.serviceDetails.additionalDetails.map((additionalItem, idx) => {
+              return (
+                <li key={'additional-details-' + idx} id={'additional-details-' + idx}>
+                  <span>{additionalItem.title}</span>
+                  {additionalItem.icon && renderAPILogo(additionalItem.icon, undefined, idx)}
+                  <TextOrLink text={additionalItem.value} urlTruncate={64} />
+                </li>
+              );
+            })}
+        </ul>
+      </div>
+    );
+
+    const serviceName = this.props.serviceDetails ? this.props.serviceDetails.service.name : 'Service';
+    return (
+      <Card id={'ServiceDescriptionCard'}>
+        <CardHeader>
+          <Title headingLevel="h5" size="lg">
+            <div key="service-icon" className={iconStyle}>
+              <Tooltip position={TooltipPosition.top} content={<>Service</>}>
+                <Badge className={'virtualitem_badge_definition'}>S</Badge>
+              </Tooltip>
```

Comment:
Please replace this with the new PFBadge:

`<PFBadge badge={PFBadges.Service} position={TooltipPosition.top} />`